docs/pages/clients/installation.md

```diff
@@ -127,6 +127,12 @@ Make the following changes to allow the Development Client to control project in
 
 There are a few more changes you can make to get the best experience, but you [can skip ahead to building](/clients/getting-started/#building-and-installing-your-first-custom-client) if you prefer.
 
+### Enable development with Expo CLI
+
+Expo CLI requires you to have the `expo` package installed so it can maintain consistent behavior in your project when new versions of the Expo SDK are released.  The package will not be used directly by you project unless you import it in your application code, which is not recommended.
+
+<InstallSection packageName="expo" cmd={["npm install expo --save-dev"]} hideBareInstructions />
```

Comment:
Do we recommend npm by default across the docs rather than yarn? 🤔

Reply:
We do:
https://docs.expo.io/
https://docs.expo.io/workflow/expo-cli/

Reply:
Fair enough, though as a counterpoint, https://docs.expo.io/clients/getting-started/ 😛 FWIW, grep yields plenty of examples of each throughout the docs. I guess we are inconsistent at best.